src/pybind/mgr/dashboard/frontend/src/app/ceph/block/rbd-form/rbd-form.component.html

```diff
@@ -88,6 +88,137 @@
           </div>
         </div>
 
+        <!-- Mirroring -->
+        <div class="form-group row">
+          <div class="cd-col-form-offset">
```

Comment:
this <div> encloses mirroring stuffs and also the `Dedicated Data pool` and its related items. That affects the styling. Can you take care of it and make them separate?